.pipelines/e2e-windows.yaml

```diff
@@ -70,3 +70,16 @@ stages:
       windowsDistro: aks-windows-2022-containerd-gen2
       windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
       storageAccount: akswinstore2022gen2
+
+- stage: backfill_cleanup_outdated_resources
```

Comment:
how do you plan to run this stage? Is there any issue if it is run by multiple PRs?

Reply:
I missed the scenario for multiple pipelines to delete outdated resources at the same time. I think I can run this script in the pipeline of creating new resources, which could avoid conflict.